test/README.md

```diff
@@ -12,7 +12,7 @@ mock interfaces rather than producing side effects.
 
 Playbook engine code is better suited for integration tests.
 
-Requirements: `sudo pip install paramiko PyYAML jinja2 httplib2 passlib nose pytest mock`
+Requirements: `sudo pip install paramiko PyYAML jinja2 httplib2 passlib nose pytest pytest-xdist mock`
```

Comment:
We should probably reference https://github.com/ansible/ansible/blob/devel/test/runner/requirements/units.txt here instead of trying to keep the docs in sync with it.